src/container/srv_target.c

```diff
@@ -292,7 +292,10 @@ cont_child_aggregate(struct ds_cont_child *cont, uint64_t *msecs)
 		epoch_min = cinfo.ci_hae;
 	}
 
-	interval = crt_sec2hlc(DAOS_AGG_THRESHOLD);
+	if (unlikely(DAOS_FAIL_CHECK(DAOS_FORCE_EC_AGG)))
+		interval = 0;
+	else
+		interval = crt_sec2hlc(DAOS_AGG_THRESHOLD);
```

Comment:
I am not very sure if the DAOS_FORCE_EC_AGG can work well. 
DAOS_AGG_THRESHOLD defined to avoid touch uncommitted DTX/record too much. Although EC agg only initiated on DTX leader, but EC agg caused re-replicate, vos_obj_array_remove on other non-leader parity node, not sure if those operation will cause some trouble with following DTX commit operation.
May confirm with @Nasf-Fan ?

But maybe fine if just for simple case for test.

Reply:
If vos aggregation hit 'prepared' DTX, it will return -DER_TX_BUSY. Then vos aggregation will skip related data record. That will not cause correctness issue, but will affect the vos aggregation efficiency.